packages/react-tinacms-strapi/README.md

```diff
@@ -26,9 +26,7 @@ export default function MyApp({ Component, pageProps }) {
         apis: {
           strapi: new StrapiClient("http://localhost:1337/"),
         },
-        media: {
-          store: new StrapiMediaStore("http://localhost:1337/"),
-        },
+        media: new StrapiMediaStore("http://localhost:1337/"),
```

Comment:
I think we had some confusion about what `store` meant when implementing the `image` field in the GraphQL form [builder](https://github.com/tinacms/tina-graphql-gateway/pull/268/files#diff-084e602bd8601f418ba3b371e7f3d985817c0b70a1a2ceb70e64d8cedd8492b2R175). What's the effect of moving this up? Seems like we'll want to standardize that

Reply:
This config change was made long ago, the docs here are just out of date and I updated them while looking over the various media store docs.

IMO this is really a bad change that necessitate the creation of the `mediaOptions` namespace in CMS config (I actually noted that here: https://github.com/tinacms/tinacms/pull/1859/files#diff-dd2f5d315632b09ae87ea8636cca1060d2e72b0136520d43aacc2166f19c0ba8R79 ) but reverting back to the "old way" would actually be a breaking change now.

Reply:
The `cms.media` situation is a little confusing; the object available at `cms.media` implements `MediaStore` but actually does a little more than what is denoted by that interface, and it wraps the userland Media Store.

The Media Store (as denoted by the `MediaStore` interface) is meant to describe the methods that need to be implemented in userland (or "plugin-land") to interface with whichever media provider is being used